src/sphinx_book_theme/assets/styles/content/_margin.scss

```diff
@@ -5,14 +5,14 @@
 // So the width we want is:
 // max-width * x = 100% - max-width, so we solve for x.
 // Will be a decimal
-$content-margin-gutter: 3%;
-$content-margin-width: percentage(
-    (100% - $content-max-width) / $content-max-width
+$content-margin-gutter: 2rem;
+$content-margin-width: (
+    (88rem - $content-max-width) 
```

Comment:
we should re-use a variable here if `88rem` is used elsewhere, or document what `88rem` comes from

Reply:
have reverted this back